app/views/govuk_publishing_components/components/docs/taxonomy_navigation.yml

```diff
@@ -0,0 +1,102 @@
+name: "Taxonomy navigation"
+description: "Sidebar navigation for displaying on pages tagged to the GOV.UK taxonomy."
+body: |
+  Accepts an array of items. These items are typically the list of taxons the current
+  content page is tagged to.
+
+  Each item is a hash with a title, url, description, and list of related content
+  associated with that item.
```

Comment:
Also worth noting it's showing policies, collections, etc.

Reply:
https://github.com/alphagov/govuk_publishing_components/pull/253/files#diff-da8a7099ce702fe7841db94b111e1489R14